### PR TITLE
FE admin claims

### PR DESCRIPTION
**Pull request by asmega**
# Context

- This updates tests to including showing FE claims in back office

### CODE REVIEW

File: app/models/claim.rb [hunk 1]
```diff
@@ -396,7 +396,7 @@ def important_notes
   end
 
   def award_amount_with_topups
-    topups.sum(:award_amount) + award_amount
+    topups.sum(:award_amount) + (award_amount || 0)
```
Comment (asmega) [inline app/models/claim.rb:399]: feeling ambivalent about this one

File: app/models/payroll_run.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@ def self.create_with_claims!(claims, topups, attrs = {})
           # associates the payment to the Topup, so we know it's payrolled
           group_topups = grouped_items.select { |i| i.is_a?(Topup) }
 
-          award_amount = grouped_items.sum(&:award_amount)
+          award_amount = grouped_items.map(&:award_amount).compact.sum(0)
```
Comment (asmega) [inline app/models/payroll_run.rb:58]: looks ugly but fixes deprecation

File: config/locales/en.yml [hunk 3]
```diff
@@ -827,6 +827,12 @@ en:
     feedback_email: "FE-Levellingup.PremiumPayments@education.gov.uk"
     support_email_address: "FE-Levellingup.PremiumPayments@education.gov.uk"
     claim_subject: "Further education payment"
+    policy_acronym: FE
```
Comment (rjlynch) [inline config/locales/en.yml:830]: I went for "TRIP" in my PR

File: config/locales/en.yml [hunk 4]
```diff
@@ -827,6 +827,12 @@ en:
...
+    policy_short_name: Further education payments
```
Comment (rjlynch) [inline config/locales/en.yml:831]: In figma I think this is now "Targeted Retention Incentive Payment For FE Teachers"

**Review by rjlynch — APPROVED** [review]
Looks good, just left one comment about the policy short name

File: config/locales/en.yml (hunk 4 above)
Comment (asmega) [reply config/locales/en.yml:831]: it's called `policy_short_name` 😂

File: config/locales/en.yml (hunk 3 above)
Comment (asmega) [reply config/locales/en.yml:830]: i'll leave for now and sort it later

**Pull request merged**